### PR TITLE
fs-1524-qa-complete-flag

### PR DESCRIPTION
**Pull request by gio-karunakaran**

### CODE REVIEW

**Review by Adam-W1 — COMMENTED** [review]
looks good :) you might need to add 'is_qa_complete' to the components.yml Flag response for when APIs are talking

**Review by Adam-W1 — COMMENTED** [review]
maybe a test for this also to get coverage

File: db/queries/flags/queries.py [hunk 1]
```diff
@@ -47,6 +47,18 @@ def retrieve_flag_for_application(application_id: str) -> dict:
...
+def find_qa_complete_flag(application_id):
+    qa_complete_flag = (
+        Flag.query.filter(
+            Flag.application_id == application_id,
+            Flag.flag_type == "QA_COMPLETED"
+        )
+        .first()
+    )
+
+    return {"is_qa_complete": True if qa_complete_flag else False}
```
Comment (gidsg) [inline db/queries/flags/queries.py:59]: can simplify logic
```suggestion
    return {"is_qa_complete": bool(qa_complete_flag)}
```

Comment (gio-karunakaran) [reply db/queries/flags/queries.py:59]: love it!

**Pull request merged**